RecoEgamma/PhotonIdentification/src/PhotonMIPHaloTagger.cc

```diff
@@ -290,17 +290,11 @@ std::vector<double> PhotonMIPHaloTagger::GetMipTrailFit(const reco::Photon* phot
            
 
         //Lets Initialize them first for each iteration 
-        res    = 0.0;
-        res_sq = 0.0;
-        wt     = 0.0; 
```

Comment:
of course it is useful to identify and remove any costly dead assignment ```log(sqrt(exp(sin(myfunc(nonCont/anotherNonConst))))```
(still a good compiler should optimixe it away. Unfortunately myfunc may have side effects
so it will not (and I am not sure the static analyser will report it...)